src/components/Modals/ConvertToMaticManaModal/ConvertToMaticManaModal.container.tsx

```diff
@@ -0,0 +1,41 @@
+import { connect } from 'react-redux'
+import { isLoadingType } from 'decentraland-dapps/dist/modules/loading/selectors'
+import {
+  approveManaRequest,
+  APPROVE_MANA_REQUEST,
+  depositManaRequest,
+  DEPOSIT_MANA_REQUEST,
+  fetchManaPriceRequest,
+  getApprovedManaRequest,
+  GET_APPROVED_MANA_REQUEST,
+  SEND_MANA_REQUEST,
+} from '../../../modules/mana/actions'
+import { RootState } from '../../../modules/reducer'
+import {
+  getManaPrice,
+  getLoading,
+  getAllowance,
+  isWaitingApprovement,
+} from '../../../modules/mana/selectors'
+import { MapDispatch, MapDispatchProps } from './ConvertToMaticManaModal.types'
+import ConvertToMaticManaModal from './ConvertToMaticManaModal'
+
+const mapState = (state: RootState) => ({
+  allowance: getAllowance(state),
+  manaPrice: getManaPrice(state),
+  isWaitingApprovement: isWaitingApprovement(state),
```

Comment:
oops this should be `approval` rather than `approvement`, would you like to change it to `isWaitingForApproval` ?

Reply:
Fixing! thanks!